index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link href='https://fonts.googleapis.com/css?family=Inter' rel='stylesheet'>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <section class="main">
+        <header>
+            <div class="normal_button">
+                <img src="images/Icon - Menu.png" width="24px" height="24px" alt="Menu-Icon">
+            </div>
+        </header>
+        <section class="headline">
+            <h2 class="greeting">Hey there.<br>I’m Alex.</h2>
+            <h2 class="intro">I’m a software developer</h2>
+            <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+               don’t hesitate to contact me.
+            </p> 
+            <section class="social_media_icons">
+                <img class="icons"  src="images/social media icons/twitter.png" alt="twitter">
+                <img class="icons" src="images/social media icons/Linkedin icon.png" alt="Linkedin">
+                <img class="icons" src="images/social media icons/medium.png" alt="medium">
+                <img class="icons" src="images/social media icons/github.png" alt="github">
+                <img class="icons" src="images/social media icons/angellist.png" alt="angellist">
+            </section>  
```

Comment:
- [ ] It is required to use `ul` to group the list of social media icons, and `li` per icon; each icon has a link (`a` tag). Kindly look into this and fix.

Reply:
do it need to provide href link as well? @Qoosim